### ISSUE TITLE
Unable to move raid member between groups while in combat

### ISSUE THREAD

**Issue by sscl500** [siweia/NDuiClassic#104]
Hey Siweia,

I noticed that there seems to be a problem with moving raid members between groups while in combat.
Not only it creates a visual error where a raid member's name stays on top of another raid member's name that needs swapping, but also, there is no way to revert it back during combat.

I couldn't save an image during raid, but here is a sample image that other user posted with the same issue.
![image1](https://user-images.githubusercontent.com/70252032/161384623-10901992-b038-4ca5-affb-c382e80656a7.jpeg)

Is there a way to fix this?

Thanks in advance!!

**Comment by siweia** [comment]
Make sure the issue exists with NDui enabled only.

**Comment by sscl500** [comment]
Yes - same issue exsits with NDui enabled only

**Comment by siweia** [comment]
I don't believe you. It seems like you are playing classic wow, not tbc.
In 60s, the classcolor for Shaman should be the same as Paladin pink.

**Comment by sscl500** [comment]
Yes, I am playing wow tbc, not classic - hence, the shaman colour is blue and not pink, like what you mentioned.
And the image, as I already mentioned, is a sample I got from google to explain the same issue that I am having as per the image.

**Comment by siweia** [comment]
If you insist the issue caused by NDui iteself, then show me with screenshotsor anything that can prove it.
![112233](https://user-images.githubusercontent.com/1502922/161413219-70958419-fb8a-4f3f-9b8e-452a28ea02d4.gif)

**Comment by sscl500** [comment]
Bear with me since I have a raid tomorrow. I will make a screenshot and upload it here

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by sscl500** [comment]
Hey Siweia,

So, I raided the black temple yesterday, and I noticed that I didn't had any issues with moving people around during combat.
However, as I will be raiding again today and tomorrow, let me try again and see if yesterday was just a fluke or not.

Let me get back soon!

Thanks,

**Comment by sscl500** [comment]
Hey Siweia,

After testing in 3 different raids, I did not come across any problems with moving raid members around during combat.
So, I wanted to let you know that I don't think that this problem is resolved at this stage.
However, if the problem persists again, I will let you know.

Thanks for your prompt response! :)

Thanks!

**Comment by siweia** [comment]
No worries. I would close this for now, let me know it the issue may exist in the future.
